includes/class-content-toc.php

```diff
@@ -433,7 +433,7 @@ protected function insert_anchors( $post_content, $toc_items_matches ) {
 			$anchors[] = sprintf(
 				'<a name="heading-%s"%s></a>',
 				esc_attr( $key_current ),
-				$this->tag_class( $this->settings['anchor_class'] ),
+				esc_attr( $this->tag_class( $this->settings['anchor_class'] ) ),
 				$match_set[0]
```

Comment:
@johnbillion do I need to use `wp_kses_post` on `$match_set[0]`? It's the original header in the post content that I matched, not sure if I need to escape it in this case?